tests/test_crs/valid_vdbenchd.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: vdbench-benchmark
+  namespace: my-ripsaw
+spec:
```

Comment:
The CI spec file needs to contain the ES information and metadata_collection: true as it waits for the backpack pods to complete. An example from the valid_backpack.yaml in the spec section.
```
  elasticsearch:
    server: "marquez.perf.lab.eng.rdu2.redhat.com"
    port: 9200
  metadata_collection: true
```

Reply:
@Avilir you are currently failing because you have the wait for backpack command in your test script but do not have metadata collection enabled. Please add the following to the spec section.

```
  elasticsearch:
    server: "marquez.perf.lab.eng.rdu2.redhat.com"
    port: 9200
  metadata_collection: true
  metadata_targeted: false
```